### DISCUSSION TITLE
Underwater object can't move at all

### DISCUSSION THREAD

**Discussion by jialing95** [DualSPHysics/DualSPHysics#226] [DUALSPHYSICS_V5.2]

**Comment by AhmedSalih3d** [comment]
Have you ensured that you underwater object has the correct mass applied so that it would be inclined to slide down the slope?

Please input your xml as a `code snippet` to get some actual help, since it is hard to help other wise.

Kind regards

**Reply by XZX2pjq** [reply]
Hi 
I would like to know if you have been able to solve this problem? Or if you have found the cause of this problem, I hope you can share it with me.Thank you very much for your help.

**Comment by alexbexe** [comment]
Try a couple of changes:
1) increase coefh to 1.2
2) change the order you create the objects: first mkbound=1 and then water, so that water at then end to fill better

Alex

**Reply by jialing95** [reply]
Hi, Alex

I've tried some changes, but they didn't help. 

Jialing

**Reply by latticeyu** [reply]
> 尝试一些更改：
> 
> 1. 将 COEFH 增加到 1.2
> 2. 更改创建对象的顺序：首先 mkBound=1，然后 water，以便 water 更好地填充
> 
> 亚历克斯

What is the specific meaning of relativeweight in the DEM module? If you can answer this question, I will be grateful, thank you
